CMakeLists.txt

```diff
@@ -53,4 +53,11 @@ SET_TARGET_PROPERTIES(redisai PROPERTIES SUFFIX ".so")
 IF (APPLE)
     SET_TARGET_PROPERTIES(redisai PROPERTIES
         LINK_FLAGS "-undefined dynamic_lookup")
-ENDIF()
\ No newline at end of file
+ENDIF()
+
+# create ramp/redisai.so with RUNPATH set to /opt/redislabs/lib for RLEC compliance
+add_custom_command(TARGET redisai POST_BUILD
```

Comment:
Can we make these commands only run in case of a RLEC build?

Reply:
The commands create a custom redisai.so that's packed into the RAMP file. I don't think there is a use case for RAMP files outside RLEC. We can do the following:
(1) Exclude the custom build for Mac,
(2) Apply patchelf as part of RAMP file creation rather than in CMake.
I think (2) is the way to go. Also, please see my comment on the Makefile.

Reply:
I agree